### ISSUE TITLE
Cannot create new project

### ISSUE THREAD

**Issue by pitgrap** [serverless-stack/serverless-stack#1843]
I cannot create a new project, if I run the command from Homepage or README. It doesn't matter, which template I choose, I always get "Template not found" and nothing happens.

```
❯ npx create-sst my-app
Need to install the following packages:
  create-sst
Ok to proceed? (y) y
? Select a template typescript-starter
✖ Template not found at presets/typescript-starter
```

Versions:
- npm -v
8.5.5
- node -v
v16.15.0

**Comment by brifiction** [comment]
Same error output as @pitgrap , however I initiated with `yarn create sst`. 

If successful, at this time we should be scaffolding new `sst` project with:
```
    "@serverless-stack/cli": "1.2.28",
    "@serverless-stack/resources": "1.2.28",
```

**Comment by teyckmans** [comment]
Same here tried a typescript, javascript and go starters all give this error

**Comment by kdallen1** [comment]
Same issue here using node 16.13.1

For me, `npm init sst@1.2.27` is successful, whereas sst@1.2.29 and sst@1.2.28 fail.

**Comment by thdxr** [comment]
Sorry about this! It's fixed in the latest release 1.2.30
